docs/changelog/124651.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 124651
+summary: "System Data Streams Should Be Restorable"
```

Comment:
Since these go into changelogs, this should be an active sentence. Something like "Fix system data streams to be restorable from a snapshot"

Reply:
Good point, thanks!